frontend/react/src/components/layout/Part.js

```diff
@@ -6,21 +6,22 @@ import QuestionComponent from "../fields/QuestionComponent";
 
 const showPart = (context_data, programType) => {
   if (context_data &&
-      programType &&
-      _.has(context_data, "show_if_state_program_type_in") &&
-      !context_data.show_if_state_program_type_in.includes(programType)) {
+    programType &&
+    _.has(context_data, "show_if_state_program_type_in") &&
+    !context_data.show_if_state_program_type_in.includes(programType)) {
     return false;
   }
   return true;
 }
 
 const Part = ({ partId, text, title, context_data, programType }) => {
   if (showPart(context_data, programType)) {
+    // Determine Part Number from partId
+    let partNum = Number(partId.split("-").pop());
     return (
       <div id={partId}>
-        {title ? <h2>{title}</h2> : <></>}
+        <h2>Part {partNum}{title ? ": " + title : <></>}</h2>
```

Comment:
```suggestion
        <h2>Part {partNum}{title ? `: ${title"` : ''}</h2>
```
Use string interpolation, and make the whole thing a string instead of a mixed string-and-component. (`<></>` is a shortcut for React's built-in `<Fragment>` component, and I'm a little wary of mixing types.)